.github/workflows/Linting.yml

```diff
@@ -30,6 +30,22 @@ jobs:
       - name: API Check
         run: gradle apiCheck
 
+  breakingChangeDetection:
+    name: ABI Breaking Change Detection
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
This is needed for fetching tags which are used for determining previous version to compare APIs against.